lib/web_ui/flutter_js/src/canvaskit_loader.js

```diff
@@ -25,24 +25,11 @@ export const loadCanvasKit = (deps, config, browserEnvironment, canvasKitBaseUrl
       canvasKitUrl = deps.flutterTT.policy.createScriptURL(canvasKitUrl);
     }
     const wasmInstantiator = createWasmInstantiator(joinPathSegments(baseUrl, "canvaskit.wasm"));
-    const script = document.createElement("script");
-    script.src = canvasKitUrl;
-    if (config.nonce) {
-      script.nonce = config.nonce;
-    }
-    script.addEventListener("load", async () => {
-      try {
-        const canvasKit = await CanvasKitInit({
-          instantiateWasm: wasmInstantiator,
-        });
-        window.flutterCanvasKit = canvasKit;
-        resolve(canvasKit);  
-      } catch (e) {
-        reject(e);
-      }
+    const canvasKitModule = await import(canvasKitUrl);
+    window.flutterCanvasKit = await canvasKitModule.default({
```

Comment:
Is there any way of naming the export from canvaskit, so it's not `default`?

((I know you can rename the `default` when [importing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import#importing_defaults), but I'd rather not have that))

Reply:
I actually tried to find a way to do so, but I don't think emscripten gives you a way to change the name of the export.

Reply:
Everything around emscriptem configuration baffles me :'(